python/cudf/cudf/tests/test_binops.py

```diff
@@ -217,10 +217,11 @@ def test_series_compare(cmpop, obj_class, dtype):
 
 def _series_compare_nulls_typegen():
     tests = []
```

Comment:
Another way to write this would be:
```python
return [
    *combinations_with_replacement(DATETIME_TYPES, 2),
    *combinations_with_replacement(TIMEDELTA_TYPES, 2),
    *combinations_with_replacement(NUMERIC_TYPES, 2),
    *combinations_with_replacement(STRING_TYPES, 2),
]
```
However you prefer is fine.